src/main/java/com/iuc/requests/controller/UserController.java

```diff
@@ -19,15 +18,26 @@
 
     @Autowired
     private UserService userService;
+    @Autowired
+    ModelMapper modelMapper;
 
     @GetMapping("/staff")
     public StaffDto findStaffByEmail(@RequestParam(value = "email") @Email String email) {
-        return null;
+        if(email.isEmpty()) {
+            return null;
+        } else{
+            return !userService.findStaffByEmail(email).equals(null)? userService.findStaffByEmail(email): null;
+        }
+
     }
 
     @GetMapping("/staff")
     public StaffDto findStaffByMatricule(@RequestParam(value = "userRegistration") String userRegistration) {
-        return null;
+        if(userRegistration.isEmpty()) {
+            return null;
+        } else{
+            return userService.findStaffByMatricule(userRegistration).equals(null)? userService.findStaffByMatricule(userRegistration): null;
```

Comment:
What is the new? I think is the same that ``return userService.findStaffByMatricule(userRegistration)``

Reply:
The comment has been taken into consideration